lib/sensors/gyroscope.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "i2c_sensors.hpp"
+
+#include <cstdint>
+#include <optional>
+
+#include <core/logger.hpp>
+#include <io/i2c.hpp>
+
+namespace hyped::sensors {
+
+static constexpr std::uint8_t kGyroscope             = 0x69;
```

Comment:
What is `kGyroscope`? The naming of this doesn't make the purpose of this constant clear?

Reply:
`kDefaultGyroscopeAddress`